File: 5semestre/Tecnologias Moveis/TDE-Aula04/src/Ex1/Pessoa.java
package Ex1;

public class Pessoa {
    private String nome;
    private int idade;

    // Construtor
    public Pessoa(String nome, int idade) {
        this.nome = nome;
        this.idade = idade;
    }

    // Método para exibir os dados da pessoa
    public void exibirDados() {
        System.out.println("Nome: " + nome);
        System.out.println("Idade: " + idade + " anos");
    }

}

File: 5semestre/Tecnologias Moveis/TDE-aula03/src/Pessoa.kt


class Pessoa(val nome: String, val idade: Int) {
    fun caminhar() {
        println("caminhando")
    }

    fun falar() {
        println("falando")
    }
}

fun main() {
    val pessoa = Pessoa("João", 22)
    pessoa.caminhar() // Saída: caminhando
    pessoa.falar() // Saída: falando
}